### ISSUE TITLE
bug take photo in Microsoft Edge

### ISSUE THREAD

**Issue by windyvu90** [jhuckaby/webcamjs#142]
Hi! I love your js, it works all browers but Microsoft edge has small bug: when I take photo, my Pic is scale.
![screenshot 1](https://cloud.githubusercontent.com/assets/15153021/12288256/15a26ff0-ba05-11e5-83b2-6277a377d885.png)

**Comment by windyvu90** [comment]
please fix that bug! thanks all

**Comment by positlabs** [comment]
Feel free to submit a PR ;)

**Comment by osdiab** [comment]
Sounds related to #129 and #24

**Comment by julienpa** [comment]
Hello! I'm sorry to bother but I'd like to know if a fix is planned for this issue?
I understand it may not be a priority or time-consuming, but the answer would allow me to find the best solution for our app using this library.
Great work by the way, thanks :)

**Comment by davidwmurray** [comment]
I have this issue also with a HD (widescreen aspect ratio) webcam.
I think it's to do with getUserMedia constraints. Edge appears to only accept the new format...
video: { width: {min: 640}, height: {min: 480}}
as opposed to the older format...
video: { mandatory: { minWidth: 640, minHeight: 480}}
So I am using the constraints option in Webcam to get this working with Edge...
                Webcam.set({
                    width: 640,
                    height: 480,
                    image_format: 'jpeg',
                    jpeg_quality: 90,
                    constraints: { width: { min: 640 }, height: { min: 480 } }
                });
This is now working for me in Edge, as well as Firefox and Chrome (latest versions). I am not able to test on older versions.

**Comment by julienpa** [comment]
Thanks for the comment and the workaround @davidwmurray!!

**Comment by MattyBalaam** [comment]
Yes, just the ticket to fix some issues I was having @davidwmurray
